package.json

```diff
@@ -20,16 +27,25 @@
   "bugs": {
     "url": "https://github.com/surikaterna/kuery/issues"
   },
+  "prettier": "prettier-config-surikaterna",
   "devDependencies": {
-    "istanbul": "^0.3.2",
-    "jshint": "^2.5.10",
-    "mocha": "^2.0.1",
-    "should": "^4.1.0",
-    "eslint": "^2.5.3",
-    "eslint-config-surikat": "^1.1.0"
+    "@swc/cli": "^0.1.57",
+    "@swc/core": "^1.2.220",
+    "@swc/jest": "^0.2.22",
+    "@types/jest": "^28.1.6",
+    "@types/lodash": "^4.14.182",
+    "@types/node": "^14.18.22",
+    "concurrently": "^7.3.0",
+    "eslint": "^8.17.0",
+    "eslint-config-surikat": "^3.0.0",
+    "jest": "^28.1.3",
+    "prettier": "^2.7.1",
+    "prettier-config-surikaterna": "^1.0.1",
+    "rimraf": "^3.0.2",
+    "typescript": "^4.7.4"
   },
   "homepage": "https://github.com/surikaterna/kuery#readme",
   "dependencies": {
-    "lodash": "^4.8.2"
+    "lodash": "4.14"
```

Comment:
Why is the version specified like this? Upgrade it by running `npm i lodash@latest`.